interfaces/acados_matlab_octave/ocp_set_ext_fun.m

```diff
@@ -421,6 +421,7 @@
 			mex(acados_include, acados_interfaces_include, external_include, blasfeo_include,...
 				hpipm_include, acados_lib_path, acados_matlab_octave_lib_path, model_lib_path, '-lacados',...
 				 '-lhpipm', '-lblasfeo', ['-l', model_name], mex_files{1});
+			setenv('CFLAGS', '');
```

Comment:
why this?

Reply:
you are right, they are not needed anymore, was connected to the qpOASES flag issue.